kube/full/deploy.sh

```diff
@@ -5,7 +5,8 @@ set -u
 
 pushd "$(dirname "$0")"
 
-# Override to use a different Docker image version for the sidecar.
+# Override to use a different Docker image name or version for the sidecar.
+export SIDECAR_IMAGE_NAME=${SIDECAR_IMAGE_NAME:-'gcr.io/stackdriver-kubernetes-1337/stackdriver-prometheus-sidecar'}
```

Comment:
This is changing the default value in kube/patch.sh. Can you justify the change?

Reply:
I moved var over to patch.sh, so it should be backwards compatible now.